.github/workflows/lint.yml

```diff
@@ -91,11 +91,10 @@ jobs:
       - name: Checking out repository
         uses: actions/checkout@v4
 
-      - name: Install addlicense
-        run: go install github.com/google/addlicense@latest
-
       - name: Run license checks
-        run: addlicense -check -ignore '.git/**' -ignore 'build/**' -ignore 'llvm*/**' .
+        run: |
+          git fetch --no-tags --prune --depth=1 origin "${GITHUB_BASE_REF?}:${GITHUB_BASE_REF?}"
```

Comment:
Yes, but this step is still needed to get the branches. Otherwise you get an error like `fatal: ambiguous argument 'main': unknown revision or path not in the working tree.` when running `git diff` in the script (see e.g. https://github.com/openxla/stablehlo/actions/runs/7936962547/job/21673253885).